### PR TITLE
release: released 1.2 version.

### PR DESCRIPTION
**Pull request by moonming**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,13 +19,53 @@
...
 - [0.6.0](#060)
 
+## 1.2.0
+
+The 1.2 version brings many new features, including core and plugins.
+
+### Core
+- :sunrise: **support etcd cluster**. [#1283](https://github.com/apache/incubator-apisix/pull/1283)
+- used the local DNS resolver by default, which is friendly for k8s. [#1387](https://github.com/apache/incubator-apisix/pull/1387)
+- support to run `header_filter` and `body_filter` phase for global rules. [#1364](https://github.com/apache/incubator-apisix/pull/1364)
```
Comment (spacewander) [inline CHANGELOG.md:37]: Should mention the `log` phase too?

File: CHANGELOG.md [hunk 2]
```diff
@@ -19,13 +19,53 @@
...
 - [0.7.0](#070)
 - [0.6.0](#060)
 
+## 1.2.0
+
+The 1.2 version brings many new features, including core and plugins.
+
+### Core
+- :sunrise: **support etcd cluster**. [#1283](https://github.com/apache/incubator-apisix/pull/1283)
+- used the local DNS resolver by default, which is friendly for k8s. [#1387](https://github.com/apache/incubator-apisix/pull/1387)
```
Comment (juzhiyuan) [inline CHANGELOG.md:36]: using

File: CHANGELOG.md (hunk 1 above)
Comment (juzhiyuan) [inline CHANGELOG.md:37]: phases

File: CHANGELOG.md [hunk 3]
```diff
@@ -19,13 +19,53 @@
...
+- support to run `header_filter` and `body_filter` phase for global rules. [#1364](https://github.com/apache/incubator-apisix/pull/1364)
+- changed the `lua/apisix` dir to `apisix`(**not backward compatible**). [#1351](https://github.com/apache/incubator-apisix/pull/1351)
+- add dashboard as submodule. [#1360](https://github.com/apache/incubator-apisix/pull/1360)
+- allow add custom shared dict. [#1367](https://github.com/apache/incubator-apisix/pull/1367)
```
Comment (membphis) [inline CHANGELOG.md:40]: allow adding

File: CHANGELOG.md [hunk 4]
```diff
@@ -19,13 +19,53 @@
...
+
+### Plugin
+- :sunrise: **add Apache kafka plugin**. [#1312](https://github.com/apache/incubator-apisix/pull/1312)
```
Comment (membphis) [inline CHANGELOG.md:43]: kafka -> Kafka

File: CHANGELOG.md [hunk 5]
```diff
@@ -19,13 +19,53 @@
...
+- :sunrise: **add CORS plugin**. [#1327](https://github.com/apache/incubator-apisix/pull/1327)
+- :sunrise: **add tcp logger plugin**. [#1221](https://github.com/apache/incubator-apisix/pull/1221)
```
Comment (membphis) [inline CHANGELOG.md:45]: TCP

File: CHANGELOG.md [hunk 6]
```diff
@@ -19,13 +19,53 @@
...
+- :sunrise: **add udp logger plugin**. [1070](https://github.com/apache/incubator-apisix/pull/1070)
```
Comment (membphis) [inline CHANGELOG.md:46]: UDP

File: CHANGELOG.md [hunk 7]
```diff
@@ -19,13 +19,53 @@
...
+- Adding support to public key based introspection for OAuth plugin. [#1266](https://github.com/apache/incubator-apisix/pull/1266)
+- response-rewrite plugin support binary data to client by base64. [#1381](https://github.com/apache/incubator-apisix/pull/1381)
+- plugin `grpc-transcode` supports grpc deadline. [#1149](https://github.com/apache/incubator-apisix/pull/1149)
+- support password auth for limit-count-redis. [#1150](https://github.com/apache/incubator-apisix/pull/1150)
+- Zipkin plugin add service name and report local server IP. [#1386](https://github.com/apache/incubator-apisix/pull/1386)
+- add `change_pwd` and `user_info` for Wolf-Rbac plugin. [#1204](https://github.com/apache/incubator-apisix/pull/1204)
+
+### Admin API
+- :sunrise: support key-based authentication for Admin API(**not backward compatible**). [#1169](https://github.com/apache/incubator-apisix/pull/1169)
+- hide ssl private key in admin API. [#1240](https://github.com/apache/incubator-apisix/pull/1240)
```
Comment (membphis) [inline CHANGELOG.md:59]: SSL

**Comment by agile6v** [comment]
LGTM

**Comment by agile6v** [comment]
The file doc/how-to-build.md and doc/how-to-build-cn.md also need to change the number to 1.2

Thanks.

File: doc/how-to-build-cn.md [hunk 8]
```diff
@@ -136,14 +136,14 @@ apisix:
 
 ```shell
 $ curl http://127.0.0.1:9080/apisix/admin/routes?api_key=abcdefghabcdefgh -i
-HTTP/1.2 200 OK
+HTTP/1.1 200 OK
```
Comment (membphis) [inline doc/how-to-build-cn.md]: good catch

**Comment by moonming** [comment]
@apache/apisix-committers please vote for 1.2 release: https://lists.apache.org/thread.html/rc8ae3207f0aae16ebf655cf0350ce5f295cdc41c208e2ae5a77085f6%40%3Cdev.apisix.apache.org%3E

**Comment by sshniro** [comment]
We can also include the batch processor util to the core if needed: (https://github.com/apache/incubator-apisix/pull/1121)

**Pull request merged**